backend/pyproject.toml

```diff
@@ -7,14 +7,15 @@ authors = ["mleader <mleader@redhat.com>"]
 [tool.poetry.dependencies]
 python = "^3.9"
 cryptography = "^3.4.8"
-pandas = "1.2.4"
+pandas = "2.2.2"
```

Comment:
Possibly; we spent a while figuring out why the backend suddenly stopped working for @MVarshini -- it turned out to be because of the new Pandas release and the fact that she was building a container (and therefore getting the latest) while I was using a stale virtual environment directly.

It's conceivable someone else would have the same issue, but I'm not really inclined to put in the extra effort, making this dependent on another PR, unless it's really necessary and can get through quickly.